index.js

```diff
@@ -26,7 +27,23 @@ function getLoaderConfig(loaderContext) {
   return assign({}, config, query);
 }
 
-module.exports = function(source) {
+/**
+ * Returns the template source with newlines and multiple spaces removed
+ * based on the processSource config option.
+ *
+ * @param {String} source
+ * @returns {String}
+ */
+var processSource = function (source) {
+  return minify(source, {
+    collapseWhitespace: true,
+    conservativeCollapse: true,
+    removeComments: true,
+    removeAttributeQuotes: true
```

Comment:
These are fairly conservative options. More can be found here:
https://www.npmjs.com/package/html-minifier